content/docs/fundamentals/optimize_amp.md

```diff
@@ -0,0 +1,230 @@
+    <link rel="canonical" href=".">
+    <title>My AMP Page</title>
+  </head>
+  <body>
+    <h1>Hello World</h1>
+  </body>
+</html>
+[/sourcecode]
+
+Let's go through it step-by-step:
+
+1.  The first tag should be the `meta charset` tag, followed by any remaining `meta` tags.
+1.  Next, preload the AMP runtime `v0.js` `<script>` tag with `<link as=script href=https://cdn.ampproject.org/v0.js rel=preload>`. The AMP runtime should start downloading as soon as possible because the [AMP boilerplate](/docs/fundamentals/spec/amp-boilerplate) hides the document via `body { visibility:hidden }` until the AMP runtime has loaded. Preloading the AMP runtime tells the browser to download the script with a higher priority. Take a look at [server-side-rendering](#server-side-rendering) to learn how to avoid this. <amp-img src="/static/img/docs/preload_resource_priorities.png"
+         width="1230" height="1068" layout="responsive"
+         alt="Priority level changes when preload is applied">
+</amp-img>
+
+1.  If your page includes render-delaying extensions (e.g., amp-experiment, amp-dynamic-css-classes, amp-story), preload those extensions as they're required by the AMP runtime for rendering the page.  [sourcecode:html]
+<link as="script" rel="preload" href="https://cdn.ampproject.org/v0/amp-custom-css-0.1.js">
+<link as="script" rel="preload" href="https://cdn.ampproject.org/v0/amp-experiment-0.1.js">
+<link as="script" rel="preload" href="https://cdn.ampproject.org/v0/story-1.0.js">[/sourcecode]
+1.  Use [preconnect](https://www.igvita.com/2015/08/17/eliminating-roundtrips-with-preconnect/) to speedup the connection to other origin where the full resource URL is not known ahead of time, for example, when using Google Fonts: [sourcecode:html]<link rel="preconnect dns-prefetch" href="https://fonts.gstatic.com/" crossorigin>[/sourcecode]
+1.  Load the AMP runtime: [sourcecode:html]<script async src="https://cdn.ampproject.org/v0.js"></script>[/sourcecode]
+1.  Specify the `<script>` tags for [render-delaying extensions](https://github.com/ampproject/amphtml/blob/master/src/render-delaying-services.js) (e.g., amp-experiment, amp-dynamic-css-classes, and amp-story)
+1.  Specify the `<script>` tags for remaining extensions (e.g., amp-bind, ...). These extensions are not render-delaying and therefore should not be preloaded as they might take away important bandwidth for the initial render.
+1.  Specify any custom styles by using the `<style amp-custom>` tag.
+1.  Add any other tags allowed in the  `<head>` section. In particular, any external fonts should go last since they block rendering.
+1.  Finally, specify the [AMP boilerplate code](/docs/fundamentals/spec/amp-boilerplate). By putting the boilerplate code last, it prevents custom styles from accidentally overriding the boilerplate css rules.
+
+{% call callout('Tip', type='success') %}
```

Comment:
@pbakaus Does this syntax still work? Or does it need to be updated?

Reply:
Good catch, the call block here can be removed, and the sentence can just start with "Tip:", and it will be auto converted.

Reply:
That doesn't work for me:

```
1.  Finally, specify the [AMP boilerplate code](/docs/fundamentals/spec/amp-boilerplate). By putting the boilerplate code last, it prevents custom styles from accidentally overriding the boilerplate css rules.


Tip: The AMP Cache performs all these optimizations automatically (and a few more). You can use the AMP Optimizer tool to automatically perform these optimizations on your own origin.

### Preload hero images
```

Renders as normal text.

Reply:
That's weird, I'll investigate but won't block this PR.